wolfcrypt/test/test.c

```diff
@@ -34951,7 +34951,11 @@ static int pkcs7signed_run_vectors(
         #endif
 
             for (j = 0, k = 2; j < (int)sizeof(digest); j++, k += 2) {
-                XSNPRINTF((char*)&transId[k], 3, "%02x", digest[j]);
+                #if defined(WOLF_C89)
```

Comment:
Could we instead change the `XSNPRINTF` macro to use sprintf when `WOLF_C89` is defined? This would allow many of the .c code changes to be reverted.

Reply:
@dgarske I would _really_ like to avoid that.  see my earlier comments.  having `XSNPRINTF()` map to an overrun-prone implementation is a recipe for future disaster.

Reply:
@dgarske, Thanks but I agree with @douzzer.